### PR TITLE
add nutrition global component

### PR DESCRIPTION
**Pull request by volodymyrmakarchuk-okta**
## Description:
- **What's changed?** 
- **Is this PR related to a Monolith release?** 

### Resolves:

* [OKTA-403628](https://oktainc.atlassian.net/browse/OKTA-403628)

### Netlify deploy:
https://60ddee956e3f68008eb03352--reverent-murdock-829d24.netlify.app/docs/release-notes/

### CODE REVIEW

**Comment by shawyu-okta** [comment]
Thanks for the netlify deploy -- Code looks fine but I'm confirming the design and usage with the team before we approve

File: packages/@okta/vuepress-theme-prose/global-components/NutritionFacts.vue [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+    </div>
+  </div>
+</template>
+
+<script>
+/*
+Inside index.md of the page you like put <NutritionFacts></NutritionFacts>
+And special data set for that component
+ex.
+nutritionItems:
```
Comment (robertnicolo-okta) [inline packages/@okta/vuepress-theme-prose/global-components/NutritionFacts.vue:28]: It wasn't exactly clear how to add the component to a page in the commented out example. When I added the below component to a page nothing was rendered.  Is the below syntax correct?

image

File: packages/@okta/vuepress-theme-prose/global-components/NutritionFacts.vue [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+  - title: This will cover
+    bulletList:
+      - Line one
+      - Line two
+  - title: Looking for something else?
+    bulletList:
```
Comment (robertnicolo-okta) [inline packages/@okta/vuepress-theme-prose/global-components/NutritionFacts.vue:34]: Even if I have only one item, do I have to use a bullet item? Does it make sense to have one item that is in a bullet list?

**Review by robertnicolo-okta — CHANGES_REQUESTED** [review]
See my comments.

File: packages/@okta/vuepress-theme-prose/global-components/NutritionFacts.vue (hunk 1 above)
Comment (volodymyrmakarchuk-okta) [reply packages/@okta/vuepress-theme-prose/global-components/NutritionFacts.vue:28]: so not exactly, I'm going to extend an example, so this data we need to put inside a special section of an `index.md`(.md files has a special section for putting any type of data, that section wrapped by `---` sybmols) file and not to leave them inside `NutritionFacts` itself

File: packages/@okta/vuepress-theme-prose/global-components/NutritionFacts.vue (hunk 2 above)
Comment (volodymyrmakarchuk-okta) [reply packages/@okta/vuepress-theme-prose/global-components/NutritionFacts.vue:34]: could you provide an example of the data format you talking about?

Comment (robertnicolo-okta) [reply packages/@okta/vuepress-theme-prose/global-components/NutritionFacts.vue:38]: Here is an example.  In the Figma mockup below, the first row has an entry with only one row and that row has no bullet point. 

image

**Review by robertnicolo-okta — CHANGES_REQUESTED** [review]
See my latest comments.

File: packages/@okta/vuepress-theme-prose/global-components/NutritionFacts.vue (hunk 2 above)
Comment (volodymyrmakarchuk-okta) [reply packages/@okta/vuepress-theme-prose/global-components/NutritionFacts.vue:38]: well spotted, now I see,  I will extend the component with special logic for this case

Comment (volodymyrmakarchuk-okta) [reply packages/@okta/vuepress-theme-prose/global-components/NutritionFacts.vue:39]: https://60ddee956e3f68008eb03352--reverent-murdock-829d24.netlify.app/docs/release-notes/

Comment (robertnicolo-okta) [reply packages/@okta/vuepress-theme-prose/global-components/NutritionFacts.vue:39]: I was able to set up the nutrition facts on a page but need some guidance on how to set one nutrition fact with only one item with no bullet point.  The example you give in the component all show nutrition facts with multiple items with bullets. Can you give an example with a single item with no bullet point?

**Review by robertnicolo-okta — CHANGES_REQUESTED** [review]

**Review by robertnicolo-okta — APPROVED** [review]
lgtm

File: packages/@okta/vuepress-theme-prose/global-components/NutritionFacts.vue (hunk 2 above)
Comment (robertnicolo-okta) [reply packages/@okta/vuepress-theme-prose/global-components/NutritionFacts.vue:39]: There is now no bullet when there is only one item which is good and I've given my LGTM.  However, I have one small nit, although the design team might have a bigger issue with it. I noticed that the text is indenting even when there is no bullet. See the screenshot below of current behavior.

image

Below are comparison screenshots showing the current and expected behavior.

image

@albertchen-okta

Comment (volodymyrmakarchuk-okta) [reply packages/@okta/vuepress-theme-prose/global-components/NutritionFacts.vue:39]: here screenshot from the latest commit
nutrition

**Pull request merged**